index.js

```diff
@@ -44,16 +46,22 @@ module.exports = {
           replacement: "import lodash from 'lodash'\nconst isPlainObject = lodash.isPlainObject"
         }
       ]
-    })
+    });
+
+    reduxTree = esTranspiler(reduxTree, {
```

Comment:
@rwjblue I took a minute to implement this and have a few more questions

In the 2nd let statement you used addon.buildBabelOptions but what is addon? I assume this is the babelAddon you pulled from the line above it ? but wanted to confirm

Next, the babel.transpileTree line itself- what is babel here? If i import babel using `babel-core` I get an error. What did you intend this to be exactly?

Thanks for the education here ;)

```js
    let babelAddon = this.addons.find(addon => addon.name === 'ember-cli-babel');
    let babelOptions = babelAddon.buildBabelOptions({
      babel: {
        plugins: [
          [require('babel-plugin-transform-object-rest-spread').default]
        ]
      }
    });

    reduxTree = babel.transpileTree(reduxTree, babelOptions);
```